builder_common/test/test_unit_queries.py

```diff
@@ -14,13 +15,13 @@ def connect_cb():
     # noinspection PyBroadException
     try:
         try:
```

Comment:
This is a sort of singleton pattern, i.e. if there isn't a connection it creates one and returns it and if there is one already it just returns the one that exists. In my opinion this is easier to see with an embedded try catch rather than raising a specifc exception to be caught in a special except statement. I just prefer this way.